graphics.tex

```diff
@@ -308,16 +306,22 @@ \subsection*{VIC IV organization and memory access basics}
 
 If you want to know more about memory management in general please have a look at \bookvref{cha:memory-map}. In BASIC - sorry to say - the situation gets even more complex. Right after turning on the computer you are in Bank 128 which is a special bank with a special memory configuration. You can use the command BANK to switch between banks. A problem here is, that you can easily lose track of what is in which bank or in which bank you are currently working. It's actually a bit easier when working in machine language. For more information about the BANK-command and banks in general, please look into \bookvref{bankcommand}. Please keep in mind that you need to know about banks before you can safely use PEEK, POKE or SYS. Otherwise you may wonder why your program does not behave as expected. We will come back to this later. Back to graphics. 
 
-Let's begin to write a simple BASIC program to show hires bitmap graphics. If you followed this chapter carefully up to this point, you should have no trouble understanding the following lines of code:
+Let's begin to write a simple BASIC program to show hires bitmap graphics. We start by using Bank 128, which is active when you turn on your MEGA65. If you followed this chapter carefully up to this point, you should have no trouble understanding the following lines of code:
 
 \begin{screenoutput}
 10 POKE $D020,0 : POKE $D021,0          :REM MAKE SCREEN BLACK
-20 POKE $D031, PEEK($D031) AND 127      :REM SET RESOLUTION
-30 POKE $DD02, 3                        :REM MAKE VIRTUAL CIA-2 READ- AND WRITEABLE 
-40 POKE $$DD00, PEEK($DD00) AND 252 OR 2:REM SELECT VIDEO BANK 1
+20 POKE $DD02, 63                       :REM MAKE VIRTUAL CIA-2 READ- AND WRITEABLE 
+30 POKE $$DD00, PEEK($DD00) AND 252 OR 3:REM SELECT VIDEO BANK 0
+40 POKE $D031, PEEK($D031) AND 127      :REM SET RESOLUTION
 \end{screenoutput}
 
-As you have seen in the illustration on the page before, the VIC chip divides the memory into 4 video banks of 16KB each. It is important to understand that you do not use a register of the VIC to define the video bank inside the (memory-)bank 0, instead you use the data bits 0 and 1 of \$DD00 or 56576 decimal. This used to be a register of the CIA-2 chip on the Commodore 64 but in your MEGA65 it is implemented as a virtual register. The "preparation" needed to set the video bank happens in line 30 while in the following line the video bank itself is set. By the way, when you convert the decimal value 3 (line 30) into its binary format, it is 00000011 and proves: the data bits 0 and 1 are set to 1, so reading and writing is activated.
+As you have seen in the illustration on the page before, the VIC chip divides the memory into 4 video banks of 16KB each.
```

Comment:
Try to preserve whitespace layout of the source so it's easier to find the actual changes in the pull request. (We should decide whether to prefer single-line paragraphs vs. hard-wrapped paragraphs, but regardless of what we decide we should make that change comprehensively in a single change without other changes.)